lib/endpoint/find-accept-header.js

```diff
@@ -0,0 +1,28 @@
+module.exports = findAcceptHeader
+
+function findAcceptHeader (state) {
+  const previewBlocks = state.blocks.filter(block => block.type === 'preview')
+
+  state.routes.forEach(route => {
+    // TODO: handle optional preview blocks
```

Comment:
I think this comment is obsolete and can be removed

Reply:
currently, there are 73 routes on `api.github.com` that come through with `previewBlocks` that have `required: false`

- https://developer.github.com/v3/apps/installations/#list-repositories
- https://developer.github.com/v3/issues/#list-issues
- https://developer.github.com/v3/orgs/#get-an-organization
- https://developer.github.com/v3/orgs/#edit-an-organization
- https://developer.github.com/v3/orgs/members/#list-pending-organization-invitations
- https://developer.github.com/v3/orgs/members/#create-organization-invitation
- https://developer.github.com/v3/orgs/members/#list-organization-invitation-teams
- https://developer.github.com/v3/repos/#list-organization-repositories
- https://developer.github.com/v3/repos/#create
- https://developer.github.com/v3/teams/#list-teams
- https://developer.github.com/v3/teams/#create-team
- https://developer.github.com/v3/repos/#edit
- https://developer.github.com/v3/repos/branches/#list-team-restrictions-of-protected-branch
- https://developer.github.com/v3/repos/branches/#replace-team-restrictions-of-protected-branch
- https://developer.github.com/v3/repos/branches/#add-team-restrictions-of-protected-branch
- https://developer.github.com/v3/repos/branches/#remove-team-restrictions-of-protected-branch
- https://developer.github.com/v3/repos/collaborators/#list-collaborators
- https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
- https://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository
- https://developer.github.com/v3/repos/comments/#get-a-single-commit-comment
- https://developer.github.com/v3/repos/comments/#list-comments-for-a-single-commit
- https://developer.github.com/v3/repos/deployments/#list-deployments
- https://developer.github.com/v3/repos/deployments/#create-a-deployment
- https://developer.github.com/v3/repos/deployments/#get-a-single-deployment
- https://developer.github.com/v3/repos/deployments/#list-deployment-statuses
- https://developer.github.com/v3/repos/deployments/#create-a-deployment-status
- https://developer.github.com/v3/repos/deployments/#get-a-single-deployment-status
- https://developer.github.com/v3/issues/#list-issues-for-a-repository
- https://developer.github.com/v3/issues/#create-an-issue
- https://developer.github.com/v3/issues/comments/#list-comments-in-a-repository
- https://developer.github.com/v3/issues/comments/#get-a-single-comment
- https://developer.github.com/v3/issues/events/#list-events-for-a-repository
- https://developer.github.com/v3/issues/events/#get-a-single-event
- https://developer.github.com/v3/issues/#get-a-single-issue
- https://developer.github.com/v3/issues/#edit-an-issue
- https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue
- https://developer.github.com/v3/issues/assignees/#remove-assignees-from-an-issue
- https://developer.github.com/v3/issues/comments/#list-comments-on-an-issue
- https://developer.github.com/v3/issues/events/#list-events-for-an-issue
- https://developer.github.com/v3/issues/labels/#list-labels-on-an-issue
- https://developer.github.com/v3/issues/labels/#add-labels-to-an-issue
- https://developer.github.com/v3/issues/labels/#replace-all-labels-for-an-issue
- https://developer.github.com/v3/issues/labels/#remove-a-label-from-an-issue
- https://developer.github.com/v3/issues/#lock-an-issue
- https://developer.github.com/v3/issues/timeline/#list-events-for-an-issue
- https://developer.github.com/v3/issues/labels/#list-all-labels-for-this-repository
- https://developer.github.com/v3/issues/labels/#create-a-label
- https://developer.github.com/v3/issues/labels/#get-a-single-label
- https://developer.github.com/v3/issues/labels/#get-labels-for-every-issue-in-a-milestone
- https://developer.github.com/v3/repos/pages/#get-information-about-a-pages-site
- https://developer.github.com/v3/repos/pages/#enable-a-pages-site
- https://developer.github.com/v3/repos/pages/#update-information-about-a-pages-site
- https://developer.github.com/v3/pulls/#list-pull-requests
- https://developer.github.com/v3/pulls/#create-a-pull-request
- https://developer.github.com/v3/pulls/comments/#list-comments-in-a-repository
- https://developer.github.com/v3/pulls/comments/#get-a-single-comment
- https://developer.github.com/v3/pulls/#get-a-single-pull-request
- https://developer.github.com/v3/pulls/#update-a-pull-request
- https://developer.github.com/v3/pulls/comments/#list-comments-on-a-pull-request
- https://developer.github.com/v3/pulls/review_requests/#create-a-review-request
- https://developer.github.com/v3/repos/#list-all-topics-for-a-repository
- https://developer.github.com/v3/repos/#replace-all-topics-for-a-repository
- https://developer.github.com/v3/search/#search-issues-and-pull-requests
- https://developer.github.com/v3/search/#search-labels
- https://developer.github.com/v3/search/#search-repositories
- https://developer.github.com/v3/search/#search-topics
- https://developer.github.com/v3/teams/#get-team
- https://developer.github.com/v3/teams/#edit-team
- https://developer.github.com/v3/teams/#delete-team
- https://developer.github.com/v3/teams/discussions/#list-discussions
- https://developer.github.com/v3/teams/discussions/#create-a-discussion
- https://developer.github.com/v3/teams/discussions/#get-a-single-discussion
- https://developer.github.com/v3/teams/discussions/

Reply:
sorry for that long comment...the non-required accept headers for those routes are being ignored right now...after a cursory look, they seem to generally be used for adding extra info in the response...which if that's true, we could just always just get everything...but some analysis may be necessary to figure out exactly how to treat these, and even ensure if they're being parsed correctly as `required: false`